README.md

```diff
@@ -63,3 +63,27 @@ Use `http://127.0.0.1:8000` as the base URL of the endpoints and navigate to `ht
 We are deploying our FastAPI instance on AWS Lambda which is triggered via an API Gateway. In order to automate the process, we are using [AWS SAM](https://www.youtube.com/watch?v=tA9IIGR6XFo&ab_channel=JavaHomeCloud), which creates the stack required for the deployment and updates it as needed with just a couple of commands and without having to do anything manually on the AWS GUI. Refer to [this](https://www.eliasbrange.dev/posts/deploy-fastapi-on-aws-part-1-lambda-api-gateway/) blog post for more details.
 
 The actual deployment happens through Github Actions. Look at [`.github/workflows/deploy_to_staging.yml`](.github/workflows/deploy_to_staging.yml) for understanding the deployment to `Staging` and [`.github/workflows/deploy_to_prod.yml`](.github/workflows/deploy_to_prod.yml) for `Production`. Make sure to set all the environment variables mentioned in [`docs/ENV.md`](docs/ENV.md) in the `Production` and `Staging` environments in your Github repository.
+
+## Tests
+
+### Installation
+
+- Install mongoengine
+
+```
+python -m pip install mongoengine
+```
+
+- Install pytest
+
+```
+pip install -U pytest
+```
+
```

Comment:
directly add them to `requirements.txt` so that they get installed directly when the original installation is happening